packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -92,6 +92,23 @@ async function convertToConfigBundle(
 		event.bundle = { ...event.bundle, modules: [] };
 	}
 
+	if (event.config.legacy?.enablePagesAssetsServiceBinding !== undefined) {
```

Comment:
should this be `==true`?

```suggestion
	if (event.config.legacy?.enablePagesAssetsServiceBinding === true) {
```

Reply:
since setting that flag to false would pass this if test

Reply:
This wasn't a boolean flag. It was an object which if set opted into using the binding with the object as config.

I realised this binding, when resolved, ended up being just a fetcher which startWorker support as-is. So I've updated this PR to pass it in directly so startWorker has no knowledge of the PagesAssetsServiceBinding and no extra options to support